source/programme-talks.rst

```diff
@@ -0,0 +1,74 @@
+Programme of talks
+==================
+
+At PyCon UK 2015, we had approximately 70 talks.
```

Comment:
Could we have a separate page in the docs about diversity of speaker line-up, linked to from this page?